src/controllers/clubsControllers.js

```diff
@@ -0,0 +1,16 @@
+import { httpCodes } from '../utils/httpCodes.js';
+import { sendResponse } from '../utils/sendResponse.js';
+import { clubService } from '../services/ClubService.js';
+
+class ClubsControllers {
+  async getAll(res) {
+    const clubs = await clubService.getAll();
+    sendResponse(res, httpCodes.SUCCESS, clubs);
```

Comment:
```suggestion
    sendResponse(res, httpCodes.SUCCESS, JSON.stringify(clubs) );
```

Здесь clubs это Json, но отправляешь на фронт ты text/plain, по этому тебе тут надо написать JSON.stringify(clubs) потому что надо твой объект перевести в строку